app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    com, first_file, second_file = command.split()
```

Comment:
"Make sure you check that the cp input command is correct." it is from the **checklist**